src/pytest_replay/__init__.py

```diff
@@ -106,22 +106,23 @@ def pytest_collection_modifyitems(self, items, config):
 
         with open(replay_file, "r", encoding="UTF-8") as f:
             all_lines = f.readlines()
-            nodeids = {
+            nodeids = dict.fromkeys(
```

Comment:
Any reason for this to be a `dict`? Seems like it could be just a `list`, given we just iterate over it, in order.

Reply:
All thanks!